test/scenarios/17. settings/advanced/show-test-networks.csv

```diff
@@ -0,0 +1,12 @@
+Advanced settings: Check “Show test networks” toggle works correctly,,,,
+Step,Test Steps,Test Data,Expected Result,Notes
+1,Open the extension.,,The Welcome Back screen is shown,
+2,Proceed to Unlock the wallet.,password (8 characters min),,
+3,Click on the right corner setting(three dots) and click on the Settings from menu.,,Settings page appears,
+4,Click on the Advanced tab.,,Advanced settings page appears,
+5,Validate that the 'Show test networks' is default to off.,,The toggle is off by default,
+6,Toggle the 'Show test networks' switch.,,The switch is turned on as expected,
+7,Click on the 'Ethereum Mainnet' drop down on the left top,,Select a network dialog box appears,
+8,Validate that the 'Show test networks' toggle is selected by default.,,The 'Show test networks' toggle is selected by default,
+9,"Validate that the test networks - 'Goerli',  'Sepolia' and 'Linea Goerli' appear",,The test networks is shown as expected,
+10,Toggle the 'Show test networks' switch in the select network dialog.,,The switch is turned off as expected in the settings page,
```

Comment:
We should add an assertion that the test networks are no longer presented in the Network selection drop down, when the toggle is turned off.

Reply:
ok sounds good, will include